src/dotnet-uninstall/Windows/ListCommandExec.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine.Rendering.Views;
+using System.Linq;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Commands;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Windows
+{
+    internal static class ListCommandExec
+    {
+        public static readonly IEnumerable<BundleTypePrintInfo> SupportedBundleTypes =
+            new BundleTypePrintInfo[]
+            {
+                new BundleTypePrintInfo<SdkVersion>(LocalizableStrings.ListCommandSdkHeader),
+                new BundleTypePrintInfo<RuntimeVersion>(LocalizableStrings.ListCommandRuntimeHeader)
+            };
+
+        public static GridView GetGridView(IList<Bundle> bundles)
+        {
+            var gridView = new GridView();
+
+            gridView.SetColumns(Enumerable.Repeat(ColumnDefinition.SizeToContent(), 3).ToArray());
+            gridView.SetRows(Enumerable.Repeat(RowDefinition.SizeToContent(), Math.Max(bundles.Count, 1)).ToArray());
+
+            foreach (var (bundle, index) in bundles.Select((bundle, index) => (bundle, index)))
+            {
+                gridView.SetChild(new ContentView(string.Empty), 0, index);
+                gridView.SetChild(new ContentView(bundle.Version.ToString()), 1, index);
+                gridView.SetChild(new ContentView($"\"{bundle.DisplayName}\""), 2, index);
```

Comment:
I know this is what we had before, but is there a reason to quote the display name in the output?

Reply:
I asked @KathleenDollard whether we should quote the display name, because otherwise the whitespace in the display name may be confusing together with column separators. I remember Kathleen thinks it is good to have double quotes around the display name.

Reply:
Its a little weird to me in a grid view since this text is in a column, but I'm not hung up on this.